ts/utils.ts

```diff
@@ -147,3 +147,11 @@ export function prettyHex(hex: string) {
     const hexNo0x = hex.slice(0, 2) == "0x" ? hex.slice(2) : hex;
     return `${hexNo0x.slice(0, 6)}…${hexNo0x.slice(-6)}`;
 }
+
+export function chunkArray<T>(arr: T[], chunkSize: number): T[][] {
+    const chunkArr = [];
+    while (arr.length) {
+        chunkArr.push(arr.splice(0, chunkSize));
+    }
+    return chunkArr;
+}
```

Comment:
If we want a more battle hardened version of this, we can bring in `lodash`'s [_.chunk](https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L6839)

Reply:
Switched to lodash.